system-contracts/contracts/EvmInterpreter.yul

```diff
@@ -344,6 +370,45 @@ object "EVMInterpreter" {
 
                     sp := pushStackItem(sp, mulmod(a, b, N))
                 }
+                case 0x50 { // OP_POP
+                    let _y
+
+                    _y, sp := popStackItem(sp)
+
+                    evmGasLeft := chargeGas(evmGasLeft, 2)
+                }
+                case 0x51 { // OP_MLOAD
+                    let offset
+
+                    offset, sp := popStackItem(sp)
+
+                    let expansionGas := expandMemory(add(offset, 32))
+
+                    sp := pushStackItem(sp, mload(add(MEM_OFFSET_INNER(), offset)))
```

Comment:
Maybe separate this for better readability

Reply:
Done